pkgs/servers/web-apps/discourse/rubyEnv/Gemfile.lock

```diff
@@ -430,7 +435,12 @@ GEM
     zeitwerk (2.4.1)
 
 PLATFORMS
+  arm64-darwin-20
   ruby
+  x86_64-darwin-18
+  x86_64-darwin-19
+  x86_64-darwin-20
+  x86_64-linux
```

Comment:
Does this cause issues if the build platform is different than the target platform?

Reply:
I don't know, but it doesn't seem likely, seeing as it builds fine on aarch64-linux, which isn't even listed. On the other hand, the lock file upgrade doesn't do anything good either - it just causes warnings about mismatched bundler versions, so I'll just revert it.